dht/dhtcore/NodeStore.c

```diff
@@ -1771,7 +1771,9 @@ struct NodeList* NodeStore_getPeers(uint64_t label,
         }
         switch (j) {
             default: Bits_memmove(out->nodes, &out->nodes[1], (j - 1) * sizeof(char*));
+                     __attribute__ ((fallthrough)); // C and C++03
```

Comment:
Causing a build error here: https://travis-ci.org/cjdelisle/cjdns/builds/243781725#L605
You can put a Gcc_FALLTHROUGH here: https://github.com/cjdelisle/cjdns/blob/master/util/Gcc.h
and then only define it if the compiler version is accepting it.